PolyEngine/Engine/Src/ResourceManager.hpp

```diff
@@ -36,9 +38,9 @@ namespace Poly
 	{
 	public:
 		//------------------------------------------------------------------------------
-		static T* Load(const String& relativePath, bool absolute = true)
+		static T* Load(const String& path, bool isAbsolute = true, eResourceSource source = eResourceSource::ENGINE)
```

Comment:
Maybe we can remove `isAbsolute` and replace it with `eResourceSource::ABSOLUTE` ? Then the name of the function `LoadTextFileRelative` should be changed to something else, i.e.: `LoadTextFileResource`.